Level Solutions/22280.0.txt

```diff
@@ -0,0 +1,43 @@
+@0
+
+
+@1
+MOV UP, RIGHT
+
+#if the number is a then move the number left/down and if it is B move rightm down and then nullify if it is one and not the other.

```

Comment:
This comment is a little unclear -- needs re-wording to explain what's going on more clearly